pkg/chunk/cache/fifo_cache.go

```diff
@@ -71,27 +71,30 @@ var (
 
 // FifoCacheConfig holds config for the FifoCache.
 type FifoCacheConfig struct {
-	Size     int           `yaml:"size"`
+	MaxSize  int           `yaml:"max_size_bytes"`
+	MaxCount int           `yaml:"size"`
```

Comment:
We may call it `MaxSizeItems` given it's the max cache size measured as max number of items